src/components/settings/PresetButtons.svelte

```diff
@@ -0,0 +1,56 @@
+<script>
+  import {
+    captionDuration,
+    captionFontSize,
+    videoSideSetting,
+    chatZoom,
+    livetlFontSize,
+    textDirection,
+    chatSplit,
+    autoVertical,
+    autoLaunchMode,
+    activePreset,
+    presets
+  } from '../../js/store';
+  import { importStores } from '../../js/storage';
+  import Button from 'smelte/src/components/Button';
+
+  $: active = $activePreset;
+  $: activePreset.set(active);
+
+  /**
+  * @param {number} number
+  */
+  async function updatePreset(number) {
+    if (number !== active) {
+      importStores(JSON.stringify(presets.get()[number - 1]));
+      active = number;
+      return;
+    }
+
+    const presetData = presets.get();
+    presetData[number - 1] = {
+      captionDuration: captionDuration.get(),
+      captionFontSize: captionFontSize.get(),
+      videoSideSetting: videoSideSetting.get(),
+      chatZoom: chatZoom.get(),
+      livetlFontSize: livetlFontSize.get(),
+      textDirection: textDirection.get(),
+      chatSplit: chatSplit.get(),
+      autoVertical: autoVertical.get(),
+      autoLaunchMode: autoLaunchMode.get()
```

Comment:
You can use `$store` instead of `store.get()` here (and line 26 as well). 

Can you also save the width and height of the splits in addition to the keyboard shortcuts and translator mode configurations in these settings?